DFC.Digital/DFC.Digital.Web.Sitefinity/App_Data/Sitefinity/Configuration/WebSecurityConfig.config.template

```diff
@@ -0,0 +1,8 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<webSecurityConfig xmlns:config="urn:telerik:sitefinity:configuration" xmlns:type="urn:telerik:sitefinity:configuration:type" config:version="12.2.7224.0">
+  <httpSecurityHeaders>
+    <responseHeaders>
+      <add value="default-src 'self'; script-src 'self' 'unsafe-inline' 'unsafe-eval' *.googleapis.com *.gstatic.com *.googletagmanager.com *.google-analytics.com apis.google.com __cdnUrl__; style-src 'self' 'unsafe-inline' 'unsafe-eval' __cdnUrl__ *.googleapis.com *.gstatic.com netdna.bootstrapcdn.com kendo.cdn.telerik.com; font-src 'self' __cdnUrl__ kendo.cdn.telerik.com netdna.bootstrapcdn.com; img-src 'self' *.gstatic.com *.googleapis.com *.google-analytics.com __cdnUrl__ https://*.dec.sitefinity.com; media-src 'self' data: blob:; child-src 'self' https://platform.twitter.com/ https://syndication.twitter.com/ https://www.youtube.com/ https://player.vimeo.com/ https://w.soundcloud.com/ apis.google.com accounts.google.com staticxx.facebook.com www.facebook.com web.facebook.com badge.stumbleupon.com; connect-src 'self' accounts.google.com https://*.dec.sitefinity.com *.mktoresp.com;" name="Content-Security-Policy" config:flags="1" />
```

Comment:
Double check these, I dont think we need facebook or youtube in here...

Reply:
Removed those references